### DISCUSSION TITLE
✅ Ping failed with ipv6 address

### DISCUSSION THREAD

**Discussion by immartian** [isontheline/pro.webssh.net#673] [General]
i tried several ipv6 addresses, which are all ping-able via desktop, receiving errors with ping tool. Does anyone share the same issue?

**Comment by isontheline** [comment]
Hello @immartian 👋

Could you post :
* the error
* the result of `ifconfig`

**Reply by immartian** [reply]
I can provide screen shots of mobile app:
![image](https://user-images.githubusercontent.com/81030/177302962-32cbe146-5a75-48f7-ab0e-584ef99b7899.png)
![image](https://user-images.githubusercontent.com/81030/177303150-1202f4ad-d47c-4997-88ff-b82393d0218b.png)

with the desktop result(within the same network):
```
ping 201:6b27:e4b8:de94:228c:d7f3:a313:a1e2

PING 201:6b27:e4b8:de94:228c:d7f3:a313:a1e2(201:6b27:e4b8:de94:228c:d7f3:a313:a1e2) 56 data bytes
64 bytes from 201:6b27:e4b8:de94:228c:d7f3:a313:a1e2: icmp_seq=2 ttl=64 time=1162 ms
64 bytes from 201:6b27:e4b8:de94:228c:d7f3:a313:a1e2: icmp_seq=3 ttl=64 time=494 ms
```

**Reply by isontheline** [reply]
Thank you a lot for your feedback 👍

I will check what's wrong asap

**Reply by isontheline** [reply]
I'm opening an issue in order to track the fix : https://github.com/isontheline/pro.webssh.net/issues/674

**Reply by isontheline** [reply]
Fixed inside #674
